src/auto-evo/RunResults.cs

```diff
@@ -584,13 +584,17 @@ public void PrintSummary(PatchMap? previousPopulations = null)
         ///   Must be final effects with <see cref="ExternalEffect.Coefficient"/> set to 1 created by
         ///   <see cref="AutoEvoRun.CalculateFinalExternalEffectSizes"/>
         /// </param>
+        /// <param name="ignoreFogOfWar">If ture ignore fog of war</param>
         /// <returns>The generated summary text</returns>
         public LocalizedStringBuilder MakeSummary(PatchMap? previousPopulations = null,
-            bool playerReadable = false, List<ExternalEffect>? effects = null)
+            bool playerReadable = false, List<ExternalEffect>? effects = null, bool ignoreFogOfWar = false)
         {
             if (previousPopulations != null && previousPopulations.CurrentPatch == null)
                 throw new ArgumentException("When previous populations is set, it must have current patch set");
 
+            if (!playerReadable)
```

Comment:
Could you add a comment here saying why this is forced to true. Future programmers will have no clue why this is done otherwise...